ColesStopAndShop/CashRegister.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ColesStopAndShop
+{
+    /// <summary>
+    /// Represents a cash register inside of a gas station of Cole's-Stop-And-Shop-franchise.
+    /// </summary>
+    public class CashRegister
+    {
+        private decimal creditBalance;
+        private decimal debitBalance;
+        private decimal salesTaxRate;
+        private int storeId;
+
+        /// <summary>
+        /// Gets the credit balance of a cash register.
+        /// </summary>
+        public decimal CreditBalance
+        {
+            get
+            {
+                return creditBalance;
+            }
+
+            private set
+            {
+                creditBalance = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets the debit balance of a cash register.
+        /// </summary>
+        public decimal DebitBalance
+        {
+            get
+            {
+                return debitBalance;
+            }
+
+            private set
+            {
+                debitBalance = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets sales tax rate.
+        /// </summary>
+        public decimal SalesTaxRate
+        {
+            get
+            {
+                return salesTaxRate;
+            }
+            private set
+            {
+                salesTaxRate = value;
+            }
+        }
+        /// <summary>
+        /// Gets the store ID.
+        /// </summary>
+        public int StoreId
+        {
+            get
+            {
+                return storeId;
+            }
+
+            private set
+            {
+                storeId = value;
+            }
+        }
+
+        /// <summary>
+        /// Initializes a cash register with a given credit balance, given debit balance, and ID number of associated store.
+        /// </summary>
+        /// <param name="creditBalance">Credit balance.</param>
+        /// <param name="debitBalance">Debit balance.</param>
+        /// <param name="storeId">Store ID.</param>

```

Comment:
- [ ] It is better to construct with a reference to the store than with a `storeID`